src/m365/entra/commands/administrativeunit/administrativeunit-add.spec.ts

```diff
@@ -182,17 +205,7 @@ describe(commands.ADMINISTRATIVEUNIT_ADD, () => {
       }
     });
 
-    await assert.rejects(command.action(logger, { options: {} } as any), new CommandError('Invalid request'));
-  });
-
-  it('passes validation when only displayName is specified', async () => {
-    const actual = await command.validate({ options: { displayName: 'European Division' } }, commandInfo);
-    assert.strictEqual(actual, true);
-  });
-
-  it('passes validation when the displayName, description and hiddenMembership are specified', async () => {
-    const actual = await command.validate({ options: { displayName: 'European Division', description: 'European Division Administration', hiddenMembership: true } }, commandInfo);
-    assert.strictEqual(actual, true);
+    await assert.rejects(command.action(logger, { options: commandOptionsSchema.parse({ displayName: 'European Division' }) } as any), new CommandError('Invalid request'));
   });
 
   it('supports specifying displayName', () => {
```

Comment:
Test cases where we check if options exist should be implemented differently with Zod:

```ts
it('supports specifying displayName', () => {
    const options = commandInfo.options;
    let containsOption = false;
    options.forEach(o => {
      if (o.long === 'displayName') {
        containsOption = true;
      }
    });
    assert(containsOption);
  });
```